server/controllers/auth/login.js

```diff
@@ -11,11 +11,12 @@ exports.login = (req, res, next) => {
   let user;
   getUserByUsername(username)
     .then(({ rows }) => {
-      if (rows && rows[0]) {
-        const { id: userId, username: userName } = rows[0];
+      if (rows[0]) {
```

Comment:
the condition like this it is big error 
you can check this: https://github.com/GSG-G7/lang-mate/pull/94#discussion_r328865145